amgut/lib/data_access/env_management.py

```diff
@@ -137,6 +138,29 @@ def make_settings_table():
         TRN.add(sql, args)
 
 
+def add_labadmin_superuser():
+    """Adds a new labadmin user names 'master' to the database and grants
+       highest privileges to this user. Password is 'password'."""
+    os.environ['PGPASSWORD'] = AMGUT_CONFIG.password
```

Comment:
Can you use the SQLConnectionHandler object that we have in the repository? That would avoid having to modify an environmental variable, plus it also will simplify error handling.

You will need to do the import inside this function, rather than at the top of the file, but I think that would be acceptable given the advantages of doing the call using the internal connection handlers.